ideascube/blog/templates/blog/content_form.html

```diff
@@ -19,13 +19,13 @@
                     {% csrf_token %}
                     <h3>{% trans 'Metadatas' %}</h3>
                     {% for field in form %}
-                        {% form_field field %}
+                        {% if field.name != 'text'  %}
+                            {% form_field field %}
+                        {% endif %}
                     {% endfor %}
                 </div>
                 <div class="col pure-u-1 pure-u-md-2-3 layout-element">
-                    <h3>{% trans 'Content' %}</h3>
-                    <label class="strip required">Content</label>
-                    <div data-editable-for="text" placeholder="{% trans 'Type your text here…' %}">{{ form.text.value|default:''|safe }}</div>
+                    {% form_field form.text %}
```

Comment:
This hunk breaks the form layout, as the two columns are not aligned any more. You probably shouldn't have removed the `<h3>...</h3>` line.